exercises/difference-of-squares/HINTS.md

```diff
@@ -1,3 +1,3 @@
 ## Hints
 For this exercise the following F# feature comes in handy:
-- [Seq.sumBy](https://msdn.microsoft.com/en-us/visualfsharpdocs/conceptual/seq.sumby%5B't,%5Eu%5D-function-%5Bfsharp%5D) is a condensed format to apply a function to a sequence and then sum the results
```

Comment:
@ErikSchierboom is there a reason that you don't think the `Seq.sumBy` hint is useful for this exercise?

Most people end up creating a square function and then doing something like this:
```
let sumOfSquares num =
    [1..num]
    |> List.sumBy square
```

The reason I had added this was I remember learning about the `sumBy` after having done it in two different lines.